definitions/pipelines/detect_variants.cwl

```diff
@@ -97,12 +100,11 @@ inputs:
     vep_to_table_fields:
         type: string[]?
         default: [HGVSc,HGVSp]
-    custom_gnomad_vcf:
-        type: File?
-        secondaryFiles: [.tbi]
-    custom_clinvar_vcf:
-        type: File?
-        secondaryFiles: [.tbi]
+    vep_custom_annotations:
```

Comment:
adding a `doc` line for these inputs with some basic description of what this is expecting would be good (and/or a pointer to the custom types file, where you could add some comments about what/how/why)

Reply:
I've added labels to the different inputs in custom type yaml. Also added a doc line to all of the `vep_custom_annotations` input lines pointing people to the custom type yaml for the input format.